cmake/configs/nuttx_auav-x21_default.cmake

```diff
@@ -13,7 +13,7 @@ set(config_module_list
 	drivers/magnetometer
 	drivers/telemetry
 
-	drivers/batt_smbus
+	drivers/battery
```

Comment:
I'm not sure we should do the rename. After all if I read this correctly we are implementing this standard?
http://sbs-forum.org/specs/sbdat110.pdf

Reply:
Each fuel gauge will have it's own special additions that warrant an individual driver.

Reply:
Why can't those be kept in individual functions?

Reply:
It helps with future maintainability and portability.